contract_id.go

```diff
@@ -24,10 +26,39 @@ func ContractIDFromString(s string) (ContractID, error) {
 	}, nil
 }
 
+func ContractIDFromSolidityAddress(s string) (ContractID, error) {
+	bytes, err := hex.DecodeString(s)
```

Comment:
Some of this logic you can de-duplicate like we have in id: https://github.com/hashgraph/hedera-sdk-go/blob/master/id.go#L9